src/args.rs

```diff
@@ -41,6 +41,10 @@ pub struct Args {
     /// User agent to be used to send requests
     #[structopt(short, long, default_value = "suckit", help="User agent to be used for sending requests")]
```

Comment:
```suggestion
    #[structopt(short, long, default_value = "suckit", help = "User agent to be used for sending requests")]
```

Not the scope of this PR, but might as well format a little while we're at it :)

Reply:
Will add